packages/app/pages/organization/course/[cid]/edit.tsx

```diff
@@ -47,19 +50,21 @@ export default function Edit(): ReactElement {
       async () =>
         await API.organizations.getCourse(organization.id, Number(cid)),
     )
+    const { data: professors } = useSWR(
+      isAdmin ? `/api/v1/organization/[oid]/get_professors` : null,
+      async () => await API.organizations.getProfessors(organization.id),
+    )
 
-    if (error) {
-      router.push('/organization/settings')
-    }
+    // if (error) {
+    //   router.push("/organization/settings");
+    // }
 
     const updateCourseAccess = async () => {
       await API.organizations
         .updateCourseAccess(organization.id, Number(cid))
         .then(() => {
           message.success('Course access was updated')
-          setTimeout(() => {
-            router.reload()
-          }, 1750)
+          router.back()
```

Comment:
Do we want them to go back to the previous page?

Reply:
I did `back()`, because the edit page is accesible from both the `MyCourses` page and admin panel.

Reply:
I don't think we should send them back to previous page, what if they want to edit more / confirm that changes applied?